### ISSUE TITLE
Directory in project can't find errors or warnings

### ISSUE THREAD

**Issue by DanielFloris** [squizlabs/PHP_CodeSniffer#2604]
I'm running phpcs on WSL 2 Ubuntu. I have 2 directories in my project, one called 'includes', one called 'admin'. If I run `phpcs -v includes` in my project directory, i get errors/warnings, but when I run `phpcs -v admin` the file list is correctly created and files seem to be read correctly, but no errors/warnings are flagged. This despite the fact I've purposely added some errors. 

Both directories are using the same standard (Wordpress standard).  Can anyone help me with whats going on here?

**Comment by jrfnl** [comment]
Not without more information. Maybe posting the custom ruleset you are using would help ?

**Comment by gsherwood** [comment]
I had a look at the Wordpress standards but couldn't see an exclude for `admin` directories or file paths. Are you able to post the output of `phpcs -v admin` so I can see the ruleset processing at the top, and the file list and error counts?

**Comment by DanielFloris** [comment]
Hi, thanks for looking at this. The output is:
```Registering sniffs in the WordPress standard... DONE (1 sniffs registered)
Creating file list... DONE (8 files in queue)
Changing into directory /c/Users/daniel/sites/app/app/public/wp-content/plugins/ot-production/admintest/api
Processing class-settings-api.php [PHP => 484 tokens in 80 lines]... DONE in 13ms (0 errors, 0 warnings)
Changing into directory /c/Users/daniel/sites/app/app/public/wp-content/plugins/ot-production/admintest
Processing class-ot-production-admin.php [PHP => 2551 tokens in 372 lines]... DONE in 43ms (0 errors, 0 warnings)
Changing into directory /c/Users/daniel/sites/app/app/public/wp-content/plugins/ot-production/admintest/css
Processing ot-production-admin.css [CSS => 16 tokens in 4 lines]... DONE in 2ms (0 errors, 0 warnings)
Changing into directory /c/Users/daniel/sites/app/app/public/wp-content/plugins/ot-production/admintest
Processing index.php [PHP => 2 tokens in 1 lines]... DONE in 1ms (0 errors, 0 warnings)
Changing into directory /c/Users/daniel/sites/app/app/public/wp-content/plugins/ot-production/admintest/js
Processing ot-production-admin.js [JS => 288 tokens in 44 lines]... DONE in 10ms (0 errors, 0 warnings)
Changing into directory /c/Users/daniel/sites/app/app/public/wp-content/plugins/ot-production/admintest/partials
Processing ot-production-admin-display.php [PHP => 60 tokens in 19 lines]... DONE in 1ms (0 errors, 0 warnings)
Processing ot-production-bulk-display.php [PHP => 60 tokens in 19 lines]... DONE in 1ms (0 errors, 0 warnings)
Processing ot-production-packing-display.php [PHP => 60 tokens in 19 lines]... DONE in 1ms (0 errors, 0 warnings)```
I tried copying the admin directory to "admintext" and running phpcs, I get the same result.

**Comment by DanielFloris** [comment]
False alarm. I'm an idiot. There weren't any errors or warnings to be found. The errors I was introducing weren't in the WordPress Standard ruleset. I must have been using a more strict ruleset previously. I've just tried to rename a file using an underscore, and the error was triggered.
